data/kafka/BIT-2023-25194.json

```diff
@@ -57,4 +55,4 @@
   ],
   "published": "2023-06-12T14:14:40.170Z",
   "modified": "2023-06-28T06:49:19.565Z"
```

Comment:
You are missing to update this field.

Reply:
done at https://github.com/bitnami/vulndb/pull/102/commits/f13e60efdd7fb34efb2ee202e636109182461911